### PR TITLE
ONL-4917 date range filter component

### PR DESCRIPTION
**Pull request by MaruaEl**
~~WIP Pr to gather feedbacks~~
~~Working on unit tests in the main time~~
~~looking at the failed build~~

Moving this branch from WIP to PR 
Running the visual regression tests for snapshots now

### CODE REVIEW

File: src/components/ec-date-range-filter/ec-date-range-filter.vue [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      >
+        <div
+          class="ec-date-range-filter__inputs-wrapper"
+          data-test="ec-date-range-filter__inputs-wrapper"
+        >
+          <ec-input-field
+            v-model="fromValueDate"
+            class="ec-date-range-filter__from-label"
+            type="date"
+            placeholder="DD/MM/YYYY"
```
Comment (matallo) [inline src/components/ec-date-range-filter/ec-date-range-filter.vue:20]: if we're not supporting yet different locales for our date fields the least we can do is being consistent how we're showing the placeholder (see capitalization) https://bitbucket.org/fxsolutions/eburyonline/src/ba86b02b8b4b249a806b314b41682158588804c5/frontend/src/views/trade-finance/submitted-requests/create-request/components/form/form.vue#lines-265

is the time to start thinking about a date field component? it has commonalities such error messages

File: src/components/ec-date-range-filter/ec-date-range-filter.vue [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+            :label="fromLabelText"
+            :error-message="errorMessageFromDate"
+            @change="onChange()"
+            @input="errorMessageFromDate = ''"
```
Comment (matallo) [inline src/components/ec-date-range-filter/ec-date-range-filter.vue:24]: not sure I find it the best way to do this

File: src/components/ec-date-range-filter/ec-date-range-filter.vue [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+export default {
+  name: 'EcDateRangeFilter',
+  components: { ecFilterPopover, EcInputField },
+  props: {
+    label: {
+      type: String,
+      required: true,
+      default: '',
+    },
+    value: { // Check we actually need this
```
Comment (matallo) [inline src/components/ec-date-range-filter/ec-date-range-filter.vue:61]: do we? AFAIK, we need the separate from and to values, and not something we'll compose for later use, probably we can get rid of this

**Review by matallo — CHANGES_REQUESTED** [review]

**Comment by matallo** [comment]
It's very subtle but the tab reflows when selecting a date and the badge appears, the rest LGTM!

**Review by matallo — APPROVED** [review]

**Comment by MaruaEl** [comment]
> It's very subtle but the tab reflows when selecting a date and the badge appears, the rest LGTM!

Thank you Carlos for checking it - I think the reflow will be fixed when we develop the badge - because the heigh and width of the badge should be 15px - so a bit smaller than the current one. Maybe I'll add a comment on the badge ticket so we'll be sure that that is done correctly

**Comment by matallo** [comment]
Fair enough!

**Review by stavros-tomas — APPROVED** [review]

**Review by jaimeperezpedra — APPROVED** [review]

**Review by mcibique — APPROVED** [review]

**Pull request merged**